src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -22,6 +23,23 @@ public class TypedParameter
         /// <param name="defaultValue">parameter default value</param>
         /// <param name="shortArgumentName">short name is used as tooltip</param>
         /// <param name="summary">parameter description</param>
+        [JsonConstructor]
+        public TypedParameter(string name, ProtoCore.Type type, string defaultValue)
```

Comment:
This is a workaround since defaultValue will not be deserialized to AssociativeNode by default. Looks like string type will also not be deserialized to ProtoCore.Type correctly here. Looking at other workaround